libcontainer/setns_init_linux.go

```diff
@@ -33,10 +33,12 @@ func (l *linuxSetnsInit) getSessionRingName() string {
 
 func (l *linuxSetnsInit) Init() error {
 	if !l.config.Config.NoNewKeyring {
-		if err := selinux.SetKeyLabel(l.config.ProcessLabel); err != nil {
```

Comment:
Another think, do you think this is intend to clear the key label of the process if `l.config.ProcessLabel == ""`?

Reply:
That was my worry as well. As far as I understand, by default the the context of the creating process is used, and when you write an empty string to `/proc/self/attr/keycreate` it means the same thing ("use the current context").

So, to my best understanding, writing an empty string and not writing anything is the same thing here.

Maybe @rhatdan will correct me here.